spec/helpers/application_helper_spec.rb

```diff
@@ -115,14 +115,14 @@
     context 'with extra info flag' do
       subject { links_to_conferences [conference], verbose: true }
 
-      it 'will not display empty brackets' do
+      xit 'will not display empty brackets' do
         expect(subject).to eql [link_to(conference.name, conference)]
       end
 
       context 'with location' do
         let(:conference) { build_stubbed :conference, location: 'Melée Island, Carribean' }
 
-        it 'will add the location in brackets' do
+        xit 'will add the location in brackets' do
           link_text = "#{conference.name} (Melée Island, Carribean)"
           expect(subject).to eql [link_to(link_text, conference)]
         end
```

Comment:
Which reminds me: are there currently records of conferences without the dates set?
I don't have access to the database. 

(As far as I can see, the input form that prepopulate the dates was present [from the beginning of the Conferences model  ](https://github.com/rails-girls-summer-of-code/rgsoc-teams/commit/9bf58943bb7bc49c52ea2569e25962b5bf81c974#diff-da8e2af5531074e5eb6389d13add128a) )